### PR TITLE
#383

### PR DESCRIPTION
**Pull request by Nadil-K**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
 		"prosemirror-view": "^1.32.7",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
+		"shiki": "^1.6.1",
```
Comment (supun-io) [inline frontend/package.json]: frontend doesn't need shiki, does it?

File: backend/artisan [hunk 2]
```diff
@@ -19,6 +19,8 @@ require __DIR__.'/vendor/autoload.php';
 
 $app = require_once __DIR__.'/bootstrap/app.php';
 
+ini_set('memory_limit', '256M'); // Increase memory limit to 256MB
```
Comment (supun-io) [inline backend/artisan]: remove this. for changing `ini` values locally, run `php -i` to find the ini files, and then change it in the file. 

in other words, in is an environment setting - the app should not hard code them.

Comment (Nadil-K) [reply backend/artisan]: fixed

File: frontend/package.json (hunk 1 above)
Comment (Nadil-K) [reply frontend/package.json]: Nope, only the backend. Removed it.

**Pull request merged**